samples/cpp/benchmark_app/main.cpp

```diff
@@ -557,6 +557,12 @@ int main(int argc, char* argv[]) {
 
         // If set batch size, disable the auto batching
         if (FLAGS_b > 0) {
+            if ((!FLAGS_hint.empty() && (FLAGS_hint == "throughput" || FLAGS_hint == "tput" ||
+                                         FLAGS_hint == "cumulative_throughput" || FLAGS_hint == "ctput")) ||
+                (FLAGS_hint.empty() &&
+                 (FLAGS_d.find("MULTI") != std::string::npos || FLAGS_d.find("AUTO") != std::string::npos))) {
+                slog::warn << "Batch size is set. Auto batching will be disabled" << slog::endl;
```

Comment:
Please remove AUTO from the list, since AUTO will set default hint as LATECNY per https://github.com/openvinotoolkit/openvino/pull/14236

Reply:
> Please remove AUTO from the list, since AUTO will set default hint as LATECNY per #14236

If `hint` command line parameter is empty the default behaviour for benchmark_app is use tput, is not it?